app/scripts/views/form.js

```diff
@@ -279,23 +299,12 @@ function (_, $, p, Validate, AuthErrors, BaseView, Tooltip,
       }
 
       var inputEls = this.$('input');
-      for (var i = 0, length = inputEls.length; i < length; ++i) {
-        var el = inputEls[i];
-        if (! this.isElementValid(el)) {
-          var fieldType = this.getElementType(el);
-
-          if (fieldType === 'email') {
-            return this.showEmailValidationError(el);
-          } else if (fieldType === 'password') {
-            return this.showPasswordValidationError(el);
-          }
-
-          // only one message at a time.
-          return;
-        }
+      var ret = showFirstError(inputEls);
+      if (ret !== false) {
```

Comment:
ret could be undefined which is falsy: However, in this case I still want to execute the else branch and call this.showValidationErrorsEnd();